src/Utilities/VulnerabilityRoutes.js

```diff
@@ -54,7 +54,7 @@ export const checkForAccountSystems = (isEdgeParityEnabled, hasEdgeDevices, hasC
     return isEdgeParityEnabled && (hasEdgeDevices || hasConventionalSystems) || hasConventionalSystems;
 };
 
-export const InsightsElement = ({ element: Element, title, globalFilterEnabled, ...elementProps }) => {
+export const InsightsElement = ({ element: Element, title, globalFilterEnabled, changeTitle = true, ...elementProps }) => {
```

Comment:
This component has a lot more things involved that federated module exposed to inventory does not need (e.g ZeroState). Let's create a custom hook that will fetch all necessary [here](https://github.com/RedHatInsights/vulnerability-ui/blob/ff5d00f04f7c194afc439465929305f22e067a38/src/Utilities/VulnerabilityRoutes.js#L70) and returns final result. This can be then reused inside InsightsElement and inside federated module exposed within this [wrapper](https://github.com/RedHatInsights/vulnerability-ui/blob/ff5d00f04f7c194afc439465929305f22e067a38/src/index.js#L11).